components/data_comps/desp/desp_comp_mod.F90

```diff
@@ -0,0 +1,720 @@
+#ifdef AIX
```

Comment:
No more AIX machines and if there are any in the future they probably don't need this.